# Contains max-value fields for each MinIMU9v3 sensor type.
uint8 linear_acceleration
uint16 angular_velocity
uint8 magnetic_field